3600/1/gems_stats.rb

```diff
@@ -0,0 +1,19 @@
+# :reek:TooManyInstanceVariables
+class Stats
+  attr_reader :gem_name, :used_by, :watches, :stars, :forks, :contributors, :issues
+
+  def initialize(opts = {})
```

Comment:
You don't need this initialize if you use it only once better implement 
`self.perform` method and send all attributes there
btw rename class, which statistic you collect here
Read about presenters or serializers